tests/test_bundle.py

```diff
@@ -46,7 +46,10 @@ async def test_deploy_istio_charms(ops_test: OpsTest):
     istio_charms = await ops_test.build_charms(f"{charms_path}-gateway", f"{charms_path}-pilot")
 
     await ops_test.model.deploy(
-        istio_charms['istio-pilot'], application_name='istio-pilot', trust=True
+        istio_charms['istio-pilot'],
+        application_name='istio-pilot',
+        config={'default-gateway': 'kubeflow-gateway'},
```

Comment:
The name could be anything really, shall we change it to "test-gateway"?

Reply:
Thanks for checking this! I rebased my branch and renamed the gateway.